docs/content/contributing/bicep/bicep-contribution-flow/_index.md

```diff
@@ -100,55 +107,122 @@ For more examples, see the below script's parameters section.
 
 {{< /expand >}}
 
-## 1. Setup your Azure test environment
+## 1. Fork the module source repository
+
+{{< hint type=tip >}}
+
+Checkout the [PowerShell Helper Script](#powershell-helper-script-to-setup-fork--ci-test-environment) that can do this step automatically for you! 👍
+
+{{< /hint >}}
 
 {{< hint type=note >}}
 
 Each time in the following sections we refer to 'your xyz', it is an indicator that you have to change something in your own environment.
 
 {{< /hint >}}
 
-{{< hint type=tip >}}
+Bicep AVM Modules (Resource, Pattern and Utility modules) will be homed in the [`Azure/bicep-registry-modules`](https://github.com/Azure/bicep-registry-modules) repository and live within an `avm` directory that will be located at the root of the repository, as per [SNFR19](/Azure-Verified-Modules/specs/shared/#id-snfr19---category-publishing---registries-targeted).
 
-Checkout the [PowerShell Helper Script](#powershell-helper-script-to-setup-fork--ci-test-environment) that can do this step automatically for you! 👍
+Module owners are expected to fork the [`Azure/bicep-registry-modules`](https://github.com/Azure/bicep-registry-modules) repository and work on a branch from within their fork, before then creating a Pull Request (PR) back into the [`Azure/bicep-registry-modules`](https://github.com/Azure/bicep-registry-modules) repository's `main` branch.
 
-{{< /hint >}}
+To do so, simply navigate to the [Public Bicep Registry](https://github.com/Azure/bicep-registry-modules) repository, select the `'Fork'` button to the top right of the UI, select where the fork should be created (i.e., the owning organization) and finally click 'Create fork'.
+
+### 1.1 Create a GitHub environment
```

Comment:
Is this environment also required without OIDC? If not, would it make sense to move the instructions into 'Option 1' below?